pkg/cloud/ratelimit.go

```diff
@@ -76,15 +81,66 @@ func (rl *AcceptRateLimiter) Accept(ctx context.Context, key *RateLimitKey) erro
 	return nil
 }
 
+// Observe does nothing.
+func (rl *AcceptRateLimiter) Observe(context.Context, error, *RateLimitKey) {
+}
+
+// ThrottlingStrategy handles delays based on feedbacks provided.
```

Comment:
I would like to review ThrottlingStrategy and StrategyRateLimiter separately in a different PR. Can you split this into a separate change?